OpenOversight/scripts/entrypoint_prod.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Compile any assets
```

Comment:
Some gaps between functionality and bash commands would make this easier to read.

Reply:
You mean just adding a line here, right?

Reply:
Yup! That's a much clearer way of putting it.